python/cuml/decomposition/pca.pyx

```diff
@@ -387,47 +390,52 @@ class PCA:
                                             self.noise_variance_)
         cdef uintptr_t trans_input_ptr = self._get_ctype_ptr(self.trans_input_)
 
+        cdef cumlHandle* h_ = <cumlHandle*><size_t>self.handle.getHandle()
```

Comment:
name `h_` could easily be `handle_` to enhance readability

Reply:
Also we can avoid this double cast by doing the cast to `<cumlHandle*>` in the function call which might be better for readability (though not a strong issue by any means if you think this is a better approach).